algo/src/main/java/org/neo4j/graphalgo/impl/pagerank/PageRank.java

```diff
@@ -227,7 +227,7 @@ private ComputeSteps createComputeSteps(
             DegreeComputer degreeComputer,
             long nodeCount) {
         if (concurrency <= 0) {
-            concurrency = Pools.DEFAULT_QUEUE_SIZE;
+            concurrency = Pools.DEFAULT_CONCURRENCY;
```

Comment:
Good point, but I'd say that refactoring the algo implementations is a bit out of scope for this PR.

Reply:
Normally, I try to mix development and refactoring together in the same PRs, in order for refactoring to happen continuously. I find that this is often good for code hygiene in case refactoring tasks are often backlogged and rarely prioritised (which isn't uncommon). 

How would you normally go about making a refactoring such as this one happen? In a follow-up PR, or written as an issue to be picked up later, or something else?